------------------------- 2023-08-09 23:50:02.597757
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7723102800>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 00:00:04.359601
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6fba1effd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 00:10:03.174365
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa627bfb1f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 00:20:03.292035
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f6b8e45b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 00:30:02.500351
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f95531ff2b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 00:40:02.763444
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f858f9c8940>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 00:50:02.690398
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f33209f0940>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 01:00:02.519549
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6da02d70d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 01:10:02.599327
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1e461ef2b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 01:20:03.143649
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcc611b160>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 01:30:02.926747
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5001efb0d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 01:40:02.491400
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5baa1f4970>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 01:50:02.398550
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fc145cbb0d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 02:00:03.380510
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdbebe13d00>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 02:10:03.017986
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ffa210e8940>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 02:20:02.806416
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdfb49f3160>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 02:30:02.900887
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb2f5cd8970>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 02:40:02.507636
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa0f121b2e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 02:50:02.367063
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f070e4e0970>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 03:00:03.383840
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3b34e1f2b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 03:10:02.975170
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feddfffb2b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 03:20:02.995144
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9bb8f0b1f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 03:30:03.110079
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ffbae603160>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 03:40:02.809404
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7effe2ae8940>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 03:50:02.615951
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa918fff2b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 04:00:03.222723
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcf93bb1c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 04:10:03.156520
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f49620e8940>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 04:20:02.740977
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fadbd01b160>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 04:30:03.438712
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe3bfcfb2b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 04:40:02.830921
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f35bfffb1f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 04:50:02.710154
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fc1a7fa8940>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 05:00:03.052286
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe5383e3160>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 05:10:02.828439
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3f6a70ffd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 05:20:03.265042
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f22b24e7160>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 05:30:02.879809
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9f514fb160>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 05:40:02.451915
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feb05afb2b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 05:50:02.544281
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d5a8d8970>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 06:00:03.743687
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f10e1bf3160>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 06:10:03.036197
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fad4c7170a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 06:20:02.657129
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa0131ff2e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 06:30:02.729400
Errror ------------ HTTPSConnectionPool(host='www.football-data.co.uk', port=443): Max retries exceeded with url: /fixtures.csv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5b23bd8940>: Failed to establish a new connection: [Errno -2] Name or service not known'))
------------------------- 2023-08-10 08:40:03.568828
------------------------- 2023-08-10 08:50:03.083343
------------------------- 2023-08-10 09:00:03.481246
------------------------- 2023-08-10 09:10:02.463459
------------------------- 2023-08-10 09:20:02.669761
------------------------- 2023-08-10 09:30:02.400401
------------------------- 2023-08-10 09:40:03.269598
------------------------- 2023-08-10 09:50:02.711188
------------------------- 2023-08-10 10:00:03.518721
------------------------- 2023-08-10 10:10:02.815891
------------------------- 2023-08-10 10:20:02.730576
------------------------- 2023-08-10 10:30:02.804757
